views/topics.go

```diff
@@ -99,6 +99,7 @@ func editTopic(w http.ResponseWriter, r *http.Request) {
 		title := r.PostFormValue("title")
 		content := r.PostFormValue("content")
 		isSticky := r.PostFormValue("is_sticky") == "1"
+		isClosed := r.PostFormValue("is_closed") == "1"
```

Comment:
You can make this `is_readonly` to stay consistent with the DB. How about adding a test?